### PR TITLE
Allow invoking server commands with multiple parameters

### PR DESCRIPTION
**Pull request by mherrmann**

### CODE REVIEW

**Comment by u-abramchuk** [comment]
Hi,

thank you for the pull request.

File: signalr/hubs/_hub.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def invoke(self, method, data):
         self.__connection.send({
             'H': self.name,
             'M': method,
-            'A': [data],
+            'A': data,
```
Comment (u-abramchuk) [inline signalr/hubs/_hub.py]: This change implies that the caller of invoke must ensure that he passes an array. Would it be better to replace `def invoke(self, method, data)` with `def invoke(self, method, *data)` instead?

Comment (mherrmann) [inline signalr/hubs/_hub.py]: That's a good idea. I'll update the PR.

Comment (mherrmann) [inline signalr/hubs/_hub.py]: PR is updated.

**Comment by u-abramchuk** [comment]
LGTM.

**Pull request merged**